FxAClient/Frontend/SignIn/FxASignInViewController.swift

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import UIKit
+import WebKit
+
+let SIGNUP_URI = "https://latest.dev.lcip.org/signup?service=sync&context=fx_desktop_v1"
```

Comment:
I keep screwing it up, but I think our consts are just CamelCase in iOS. i.e. SignupUri. You might as well make these private too (and the top one isn't used?)